WooCommerce/Classes/ViewRelated/Dashboard/JetpackConnectionPackageSites/JetpackBenefitsView.swift

```diff
@@ -78,6 +95,16 @@ struct JetpackBenefitsView: View {
 }
 
 private extension JetpackBenefitsView {
+    @MainActor
+    func fetchJetpackUser() async -> Result<JetpackUser, Error> {
```

Comment:
❓ Should we consider introducing a view model to handle the network calls?

Reply:
Added the view model in a320f4daf21f70e8e086023545a31253dd5ddc30 and unit tests in 6e65309e3733daa0dafcd187607b4186df648f9a.